pacta_data_qa.Rmd

```diff
@@ -0,0 +1,678 @@
+---
+title: "PACTA Analysis - Input Data QA"
+author: "Jacob Kastl"
+date: "`r format(Sys.time(), '%B %Y')`"
+mainfont: GT America
+params: 
+  project_code: general
+  project_location_ext: NULL
+  twodii_internal: TRUE
```

Comment:
for my own understanding... this allows for these variable to be set when this is rendered?

Reply:
Yep

Reply:
Yep